### PR TITLE
Updated Sponsors to Supporters 

### PR DESCRIPTION
**Pull request by Bitc0indad**
Changed website Sponsors to Supporters

### CODE REVIEW

**Comment by bitkarrot** [comment]
@Bitc0indad do you have a vercel deploy? wanted to see the bitcoin.astro before merging, thanks!

File: api/bitcoin_data.py
Comment (bitkarrot) [inline api/bitcoin_data.py:1]: how is this used? there is a bitcoin.astro, but the data is not being pulled from api.

Comment (Bitc0indad) [reply api/bitcoin_data.py:1]: Apologies!!! It's not styled yet and still working out the backend but we will get that resolved soon. 

https://website-bitc0indad.vercel.app/ 

Updated and deployed for inspection

**Comment by Bitc0indad** [comment]
Removed bitcoin.astro ready for review 

We will work on the bitcoin.astro later

File: api/bitcoin_data.py
Comment (bitkarrot) [reply api/bitcoin_data.py:1]: ok!

**Comment by Bitc0indad** [comment]
> Can you also remove the bitcoin.py? its not doing anything right now. Thanks

done!

**Review by bitkarrot — APPROVED** [review]

**Pull request merged**